source/sourcing_and_processing_data/processors/convert_degrees.rst

```diff
@@ -0,0 +1,51 @@
+Convert Degrees processor
+=========================
+
+This processor converts a degrees, minutes, seconds geo coordinate to a standard geo coordinate.
```

Comment:
"standard geo coordinate" could be replace by "'lat, lng' notation (which is the format expected by the platform"